catalog/admin/module/catalog-admin-module-sources/src/main/webapp/index.html

```diff
@@ -33,6 +33,9 @@
     <![endif]-->
 </head>
 <body>
+<div id="page-loading">
```

Comment:
Worked with @andrewkfiedler to cleanup the code some.  He thought this approach was easiest to implement and easiest to understand.